keda/README.md

```diff
@@ -107,7 +110,6 @@ their default values.
 | `prometheus.metricServer.podMonitor.namespace`             | Scraping namespace for metric server using podMonitor crd (prometheus operator) | ``
 | `prometheus.metricServer.podMonitor.additionalLabels`      | Additional labels to add for metric server using podMonitor crd (prometheus operator) | `{}`
 | `prometheus.operator.enabled`                              | Enable keda operator prometheus metrics expose | `false`
-| `prometheus.operator.port`                                 | HTTP port used for exposing keda operator prometheus metrics | `9022`
```

Comment:
Is this on purpose?

Reply:
Yeah. Operator metrics aren't exposed at port 9022. They're exposed at wherever `.Values.service.operator.portHttpTarget/metrics`